Makefile

```diff
@@ -7,8 +7,8 @@ testing: deps
 	mix compile --warnings-as-errors --force
 	mix format --check-formatted
 	mix credo --strict
-	MIX_ENV=test mix coveralls.json
-	mix dialyzer --halt-exit-status
+	mix coveralls.json
+	# mix dialyzer --halt-exit-status
```

Comment:
Why'd we disable this?

Reply:
No reason at all other than `WIP` and dialyzer takes a lot of time.

Reply:
Coolio! Just curious 😁